MeetupPlatformApi/MeetupPlatformApi.csproj.user

```diff
@@ -1,7 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please don't use scaffolding capabilities of Visual Studio in this project and especially production solutions. Usually scaffolding templates are outdated, poorly formatted, hard to extend and constructed in simplicity to generate in mind (good code should be easy to read or at least easy to write and extend, but nor easy to generate using automatic code generation solutions).